pyro/distributions/transforms/lower_cholesky_affine.py

```diff
@@ -0,0 +1,58 @@
+import torch
+from torch.distributions import constraints
+from torch.distributions.transforms import Transform
+
+from pyro.distributions.util import copy_docs_from
+
+
+@copy_docs_from(Transform)
+class LowerCholeskyAffine(Transform):
+    """
+    A bijection of the form :math:`\\mathbf{y} = \\mathbf{L} \\mathbf{x} + \\mathbf{r}`
+    where `\\mathbf{L}` is a lower triangular matrix and `\\mathbf{r}` is a vector.
+
+    :param loc: the fixed D-dimensional vector to shift the input by.
+    :type loc: torch.tensor
+    :param scale_tril: the D x D lower triangular matrix used in the transformation.
+    :type scale_tril: torch.tensor
+    """
+    codomain = constraints.real
+    bijective = True
+    event_dim = 1
+    volume_preserving = False
```

Comment:
This seems to be unused?

Reply:
yeah it's something stefan added thinking it might be useful later 🤷‍♂

Reply:
Oh..this seems like something Stefan has introduced in our transforms module?

Reply:
yes